### DISCUSSION TITLE
「灌水」急急急！新人第一次进绳网需要注意什么

### DISCUSSION THREAD

**Discussion by A-chllenge** [share121/inter-knot#3470] [其他]
![IMG_20180830_221854](https://github.com/user-attachments/assets/03e1b8d1-eebc-4f16-b0ec-6cb13768fa33)
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" >有没有老登带一带

**Comment by LT-UT** [comment]
永远遵守绳网规范的规则，遇到违反规则的帖子举报，评论时不能谈不好的东西
应该就可以了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
